### DISCUSSION TITLE
专业IPFS分币系统 + APP出售

### DISCUSSION THREAD

**Discussion by runtuer** [filecoin-project/community-china#184] [应用开发]
专业IPFS分币系统平台 + app全套软件出售
产品特点：
1，可支持B端
2，可支持C端
3，可按节点、平均、自定义产能方式产币
4，分币比例可调整
5、系统全法合规，不踩FL红线
6、C端模式时奖励方式多样
如需了解详情，请加V， 请备注来意
 ![avatar](http://test.fenbi.xjcc.top/assets/index/images/theseaer-wx.jpg)

**Comment by yunfens** [comment]
项目整体转让，fil机房配置，3台空调，50千瓦发电机，30千瓦ups；存储配置：6台36盘14T存储机外加5台同配置空的存储机（不带硬盘），5台封装机，每天10T的封装能力，总体3P存储，已经封装了430T，一台节点机，7张3090显卡，算力均为1t内存，16tp4510缓存盘，3090显卡adm7402，带分币系统app及源码，项目整体转让，并免费提供技术支持！机房租金3000每月，可放置6台42U机柜，并可以扩充更多！vx:186-2004-4433

**Reply by runtuer** [reply]
> 项目整体转让，fil机房配置，3台空调，50千瓦发电机，30千瓦ups；存储配置：6台36盘14T存储机外加5台同配置空的存储机（不带硬盘），5台封装机，每天10T的封装能力，总体3P存储，已经封装了430T，一台节点机，7张3090显卡，算力均为1t内存，16tp4510缓存盘，3090显卡adm7402，带分币系统app及源码，项目整体转让，并免费提供技术支持！机房租金3000每月，可放置6台42U机柜，并可以扩充更多！vx:186-2004-4433

这么久了还没转出去啊？
